### PR TITLE
ClusterLoader - extracting prometheus query execution to util

### PR DESCRIPTION
**Pull request by krzysied**
Extracting prometheus query execution from api_responsiveness_proemtheus.go to pkg/measurement/util.

### CODE REVIEW

File: clusterloader2/pkg/measurement/util/prometheus.go [hunk 1]
```diff
@@ -74,6 +88,73 @@ type promResponseData struct {
 	v model.Value
 }
 
+// ExecutePrometheusQuery executes given promethous query.
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/util/prometheus.go]: s/promethous/prometheus/

File: clusterloader2/pkg/measurement/util/prometheus.go [hunk 2]
```diff
@@ -74,6 +88,73 @@ type promResponseData struct {
...
+// If not config is provided the default one is used.
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/util/prometheus.go]: If *no* config

File: clusterloader2/pkg/measurement/util/prometheus.go [hunk 3]
```diff
@@ -74,6 +88,73 @@ type promResponseData struct {
...
+	for _, sample := range samples {
+		if !math.IsNaN(float64(sample.Value)) {
+			resultSamples = append(resultSamples, sample)
+		}
+	}
+	return resultSamples, nil
+}
+
+// PrometheusConfig contains data required to access promethous server.
+type PrometheusConfig struct {
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/util/prometheus.go]: TBH, I don't see anything here that clients of this util would like to customize in the near future. IMHO we can hide all these setting inside the implementation for now.

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/util/prometheus.go]: +1

File: clusterloader2/pkg/measurement/util/prometheus.go (hunk 1 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/util/prometheus.go]: Done.

File: clusterloader2/pkg/measurement/util/prometheus.go (hunk 2 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/util/prometheus.go]: Line removed (there is no config now).

File: clusterloader2/pkg/measurement/util/prometheus.go (hunk 3 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/util/prometheus.go]: Removed.

**Pull request merged**